apps/api/src/purchase-orders/dto/create-purchase-order.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { IsDateString, IsString, IsArray, IsNumber, ValidateNested, Min } from 'class-validator';
+import { Type, Transform } from 'class-transformer';
+
+export class PurchaseOrderLineItemDto {
+    id: number;
```

Comment:
missing @isNumber() here - if id is number. Class validator strips out any values it can't find decorators for, which can be a bit counter intuitive and hard to debug

Reply:
Good catch, fixed!